hw3/03-react/src/routes/Houses.js

```diff
@@ -0,0 +1,74 @@
+import { backgroundColors, borderColors } from './../utils/chartColors';
+import './../css/houses.css';
+import { Doughnut } from 'react-chartjs-2';
+import { Chart, ArcElement } from 'chart.js';
+Chart.register(ArcElement);
+
+function Houses(props) {
+  const { title } = props;
+  const { characters } = props;
+
+  const houses = [
+    'Targaryen',
+    'Tarly',
+    'Stark',
+    'Baratheon',
+    'Lannister',
+    'Greyjoy',
+    'Clegane',
+    'Baelish',
+    'Seaworth',
+    'Unknown',
+    'Tyrell',
+    'Free Folk',
+    'Tarth',
+    'Naathi',
+    'None',
+    'Bolton',
+    'Naharis',
+    'Lorathi',
+    'Mormont',
+    'Sparrow',
+    'Viper',
+    'Sand',
+    'Worm',
+    'Qyburn',
+    'Bronn',
+    'Corrupt Data',
+  ];
+  let count = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0];
```

Comment:
I think using a map would have made working with the data a bit easier - but I don't think you need to change it since you've already figured out the logic!

Reply:
Yeah I think that would've been a lot neater. Thanks for the tip!